src/boot_O2/rand.c

```diff
@@ -77,8 +87,8 @@ f32 Rand_ZeroOne_Variable(u32* rndNum) {
  */
 f32 Rand_Centered_Variable(u32* rndNum) {
     u32 next = (*rndNum * RAND_MULTIPLIER) + RAND_INCREMENT;
-    // clang-format off
-    *rndNum = next; sRandFloat = (next >> 9) | 0x3F800000;
-    // clang-format on
+
+    *rndNum = next;
+    sRandFloat = (next >> 9) | 0x3F800000;
```

Comment:
I think I would prefer this to have the same style as the one above (assign `next` inside the calculation), though not a huge deal.

Reply:
Yeah, I suppose that makes sense, it's just IDO being weird.